src/test/java/com/ookawara/book/application/service/BookServiceTest.java

```diff
@@ -33,4 +33,57 @@ class BookServiceTest {
         List<JoinedBook> actual = bookService.findBy("", "", null);
         assertThat(actual).isEqualTo(allBooks);
     }
+
+    @Test
+    public void カテゴリーのみ文字を指定したときその文字がカテゴリーに含まれている全ての本のデータを取得() {
```

Comment:
このままでも問題ないですが、こういう書き方もあるんだなーぐらいで🙏
```suggestion
    public void 指定した文字列を含むカテゴリーに該当する全ての本のデータを取得() {
```

Reply:
なるほど。
確かに教えていただいた方がコンパクトでわかりやすいですね！
書籍名の方も`指定した文字列を含む書籍名に該当する全ての本のデータを取得する` と書き換えることもできるということですよね！